### PR TITLE
[android] Show copyright in proper place.

### PR DESCRIPTION
**Pull request by yunikkk**
https://trello.com/c/EGWtNzEN/209-android-copyright-osm

### CODE REVIEW

File: android/src/com/mapswithme/maps/MapFragment.java [hunk 1]
```diff
@@ -48,6 +48,7 @@
   private int mWidth;
   private boolean mRequireResize;
   private boolean mEngineCreated;
+  private static boolean mWasCopyrightDisplayed;
```
Comment (trashkalmar) [inline android/src/com/mapswithme/maps/MapFragment.java]: sCopyrightWasDisplayed

Comment (yunikkk) [inline android/src/com/mapswithme/maps/MapFragment.java]: +

**Comment by rokuz** [comment]
LGTM

**Comment by yunikkk** [comment]
@trashkalmar PTAL

**Pull request merged**